packages/frontend-app-api/api-report.md

```diff
@@ -7,13 +7,16 @@ import { BackstagePlugin } from '@backstage/frontend-plugin-api';
 import { Config } from '@backstage/config';
 import { ConfigApi } from '@backstage/core-plugin-api';
 import { ExtensionDataRef } from '@backstage/frontend-plugin-api';
+import { ExtensionOverrides } from '@backstage/frontend-plugin-api';
 import { JSX as JSX_2 } from 'react';
 
 // @public (undocumented)
 export function createApp(options: {
-  plugins: BackstagePlugin[];
+  features?: (BackstagePlugin | ExtensionOverrides)[];
```

Comment:
Would that be convenient to create a `BackstageFeature` type?

Reply:
Yep though we might leave that for a followup discussion though as I'm not sure whether we should go make that an opaque type or not. On a side note I figured we might make it `FrontendPlugin` and `FrontendFeature` too

Reply:
Sounds like a good plan for a followup pull request, closing the thread 🙂

Reply:
`FrontendFeature` to match the existing `BackendFeature`!